src/classes/ui.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Cronbeat;
+
+use Cronbeat\Views\BaseView;
+use Cronbeat\Views\SetupView;
+use Cronbeat\Views\LoginView;
+
+class UI {
+    public function renderSetupForm($error = null) {
+        $view = new SetupView();
+        $view->setError($error);
+        return $view->render();
+    }
+
+    public function renderLoginForm($error = null) {
+        $view = new LoginView();
+        $view->setError($error);
+        return $view->render();
+    }
+
+}
```

Comment:
@jetbrains-junie this file is not used, except of UITest.php, remove both files and commit back to this pr.